### PR TITLE
✨ Feature: Tag releases on push to production, bump the version in package.json on staging branch

### PR DESCRIPTION
**Pull request by glenna**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+      - name: "fetch default branch"
+        run: git fetch origin ${{ github.repository.default_branch }}
+      - name: "cat package.json"
+        run: cat ./package.json
+      - name: "Setup Node.js"
+        uses: "actions/setup-node@v1"
+        with:
+          node-version: 8.10.0
```
Comment (stefanoeb) [inline .github/workflows/push.yml:25]: since the docker image has node already, I feel like this is not necessary

File: index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+const { Toolkit } = require("actions-toolkit");
```
Comment (stefanoeb) [inline index.js:1]: fancy!

File: index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+const github = require("@actions/github");
+
+// Change working directory if user defined PACKAGEJSON_DIR
+if (process.env.PACKAGEJSON_DIR) {
+  process.env.GITHUB_WORKSPACE = `${process.env.GITHUB_WORKSPACE}/${process.env.PACKAGEJSON_DIR}`;
+  process.chdir(process.env.GITHUB_WORKSPACE);
+}
+
+function getNewVersion() {
+  return execSync("npm version --git-tag-version=false patch")
```
Comment (stefanoeb) [inline index.js:12]: that's a nice trick! I'd def do some string manipulation 😂

File: .github/workflows/push.yml (hunk 1 above)
Comment (glenna) [reply .github/workflows/push.yml:25]: ohh, true!

**Pull request merged**